src/php/ext/grpc/channel.c

```diff
@@ -353,10 +459,17 @@ PHP_METHOD(Channel, __construct) {
  */
 PHP_METHOD(Channel, getTarget) {
   wrapped_grpc_channel *channel = Z_WRAPPED_GRPC_CHANNEL_P(getThis());
+  if (channel->wrapper == NULL) {
+    zend_throw_exception(spl_ce_RuntimeException,
+                         "getTarget error."
+                         "Channel is already closed.", 1 TSRMLS_CC);
+    return;
+  }
   gpr_mu_lock(&channel->wrapper->mu);
```

Comment:
Can you combine these 2 checks?

Reply:
I will remove the second check.
It is the check for the old behavior, which [shared channel will destroy the real channel and make it to null](https://github.com/grpc/grpc/pull/15218/files/591fd4657130e335e0ae91747b2190498752a3e7#diff-f942bdda09aac4a837cef783506850f0R514)